### ISSUE TITLE
create element with javascript

### ISSUE THREAD

**Issue by ruhancarloss** [riophae/vue-treeselect#314]
I have a button 'Add', this button event create new line in my table. 

In the new line, i need create treeselect dynimac with javascript, this is possible?

**Comment by jonagoldman** [comment]
Yes, you can use Vue and [dynamically create a new component](https://vuejs.org/v2/guide/components.html#Dynamic-Components). You can also dynamically generate template or anything you need, just use Vue as regular.
